mono/metadata/exception-internals.h

```diff
@@ -74,4 +78,10 @@ mono_exception_new_invalid_operation (const char *msg, MonoError *error);
 MonoExceptionHandle
 mono_error_convert_to_exception_handle (MonoError *error);
 
+MonoExceptionHandle
```

Comment:
The pattern had been “new” not “get”, to allocate exceptions and return a handle to them. Oops.